types/setting.go

```diff
@@ -140,12 +137,16 @@ const (
 	SettingNameDefaultMinNumberOfBackingImageCopies                     = SettingName("default-min-number-of-backing-image-copies")
 	SettingNameBackupExecutionTimeout                                   = SettingName("backup-execution-timeout")
 	SettingNameRWXVolumeFastFailover                                    = SettingName("rwx-volume-fast-failover")
+	// These three backup target settings are used in the "longhorn-default-setting" ConfigMap
```

Comment:
To prevent any confusion, we might need to decouple these settings from global settings in chart values.yaml.

Reply:
cc @mantissahz